src/WebUI/Components/KubeSummary.tsx

```diff
@@ -21,16 +20,27 @@ import "azure-devops-ui/Label.scss";
 import { DaemonSetListingComponent } from "./DaemonSetListingComponent";
 import { StatefulSetListingComponent } from "./StatefulSetListingComponent";
 import { PodsComponent } from "./PodsComponent";
+import { Filter, IFilterState, FILTER_CHANGE_EVENT, IFilterItemState } from "azure-devops-ui/Utilities/Filter";
+import { KubeResourceType } from "../../Contracts/KubeServiceBase";
+import { FilterComponent } from "./FilterComponent";
+import { Tab, TabBar, TabContent } from "azure-devops-ui/Tabs";
+import { ObservableValue } from "azure-devops-ui/Core/Observable";
+import { HeaderCommandBarWithFilter } from 'azure-devops-ui/HeaderCommandBar';
 
 const workloadsPivotItemKey: string = "workloads";
 const servicesPivotItemKey: string = "services";
+const filterToggled = new ObservableValue<boolean>(false);
 
 export interface IKubernetesContainerState extends IKubernetesSummary {
     selectedKey?: string;
     showSummary?: boolean;
     showDeployment?: boolean;
     showService?: boolean;
     selectedItem?: any;
+    filter:Filter;

```

Comment:
in future when we make the code clean, we should make these to individual components.

following in workloads
filter:Filter;
workloadsFilterState?:IFilterState;

following in services
    svcFilter: Filter;    
    svcFilterState?: IFilterState;

would you write a TODO please?

Reply:
Done.